src/java.desktop/share/classes/javax/swing/JList.java

```diff
@@ -3228,6 +3229,11 @@ protected class AccessibleJListChild extends AccessibleContext
             private ListModel<E> listModel;
             private ListCellRenderer<? super E> cellRenderer = null;
 
+              static {
+                  AccessibleComponentAccessor.addAccessor(c ->
+                          c instanceof JList.AccessibleJList.AccessibleJListChild ? ((JList.AccessibleJList.AccessibleJListChild) c).getCurrentComponent() : null);
```

Comment:
Please split the long lines to have 80 chars per line.

Reply:
Donw